src/mmw/js/src/modeling/tr55/quality/views.js

```diff
@@ -86,11 +86,28 @@ var ResultView = Marionette.LayoutView.extend({
 
     downloadCSV: function() {
         var data = this.model.get('result').quality,
+            aoiVolumeModel = new AoiVolumeModel({
+                areaOfInterest: this.options.areaOfInterest }),
             prefix = 'tr55_water_quality_',
             timestamp = new Date().toISOString(),
             filename = prefix + timestamp;
 
-        utils.downloadDataCSV(data, filename);
+        var adjustedData =  _.map(data, function(row) {
```

Comment:
I kept this fn here rather than moving it to a constants or utils file since the calculations need some methods from the `aoiVolumeModel` available in this file.